.github/workflows/ci-check.yml

```diff
@@ -1,92 +1,61 @@
         with:
-            arguments: generateTestTasksJson
-
-    -   id: setup-matrix
+          arguments: generateTestTasksJson
+      - id: setup-matrix
         run: echo "::set-output name=matrix::$(cat build/build-resources/androidTestTasks.json)"
-
-    -   name: debug
+      - name: debug
         run: echo ${{ steps.setup-matrix.outputs.matrix }}
-
     outputs:
-        matrix: ${{ steps.setup-matrix.outputs.matrix }}
+      matrix: ${{ steps.setup-matrix.outputs.matrix }}
 
   sanity_check:
+    name: Sanity check
     runs-on: ubuntu-latest
-
     steps:
-    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-    -   uses: actions/checkout@v1
-
-    # Use Java 8
-    -   uses: actions/setup-java@v1
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v2
         with:
-            java-version: 11
-
-    # Gradle build
-    -   uses: gradle/gradle-build-action@v2
-        env:
-          GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GE_ACCESS_TOKEN }}
+          java-version: 11
+      - uses: gradle/gradle-build-action@v2
         with:
-            arguments: sanityCheck
+          arguments: sanityCheck
 
   android_version_tests:
-    needs: [generate_versions, sanity_check]
-
-    # The type of runner that the job will run on
+    name: Android version tests
+    needs: [ generate_versions, sanity_check ]
     runs-on: ubuntu-latest
-
     strategy:
       fail-fast: false
       matrix:
         androidTestTask: ${{ fromJson(needs.generate_versions.outputs.matrix) }}
-
-    # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
-    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-    -   uses: actions/checkout@v1
-
-    # Patch issue in platform-tools 31.0.3 where platform-tools/api/api-versions.xml is missing (see https://issuetracker.google.com/issues/195445762)
-    -   name: Patch api-versions
+      - uses: actions/checkout@v2
+      # Patch issue in platform-tools 31.0.3 where platform-tools/api/api-versions.xml is missing (see https://issuetracker.google.com/issues/195445762)
+      - name: Patch api-versions
         run: sudo test -f $ANDROID_HOME/platform-tools/api/api-versions.xml || (sudo mkdir $ANDROID_HOME/platform-tools/api && sudo cp .github/api-versions.xml $ANDROID_HOME/platform-tools/api/api-versions.xml)
-
-    # List all installed Android SDK packages (for troubleshooting)
-    -   name: List Android Packages
+      - name: List Android Packages
         run: sudo $ANDROID_HOME/tools/bin/sdkmanager --list | sed -n '/Available Packages/q;p'
-
-    # Use Java 8
-    -   uses: actions/setup-java@v1
+      - uses: actions/setup-java@v2
         with:
-            java-version: 8
-
-    # Gradle build
-    -   uses: gradle/gradle-build-action@v2
-        env:
-            GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GE_ACCESS_TOKEN }}
+          java-version: 11
```

Comment:
Is the move from Java 8 to Java 11 intentional?

Reply:
I assumed that it is.  We used to build with JDK8 because older versions of AGP required it.  So, we would build with JDK8, test those versions with JDK8, and test against newer versions of AGP using JDK11.  With the work to drop support for older versions of AGP we did recently, we now only support versions that require JDK11 and up.  As far as I can tell, there's no reason to build with JDK8 anymore.

Reply:
Good question. Yes, this is intentional. AGP 7+ only supports Java 11+.
The build also has the toolchain version set to 11 https://github.com/gradle/android-cache-fix-gradle-plugin/blob/main/build.gradle#L61